src/scss/layout/_header.scss

```diff
@@ -1,3 +1,68 @@
-.header {
-  height: 90px;
+@mixin  fontStyle ($color, $fontSize) {
+    color: $color;
+    font-size: $fontSize;
+    font-family: Helvetica;
+    font-style: normal;
+    font-weight: bold;
 }
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    height: 80px;
+    background: #FFFFFF;
+    padding: 30px 90px 30px 80px;
+}
+
+li, a, .btn {
+    @include fontStyle(#2c8eff, 16px);
+    line-height: 18px;
+    text-decoration: none;
+}
+
```

Comment:
Keep spacing between selectors consistent. Some of the have 1, others 2 and 3 spaces

Reply:
Thanks you. Done.